### PR TITLE
Fix(kclvm-runner): Fixed file locks in kclvm-runner.

### PR DESCRIPTION
**Pull request by zong-zhe**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

fix issue https://github.com/KusionStack/KCLVM/issues/213.

#### 2. What is the scope of this PR (e.g. component or file name):

kclvm-runner

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other

The operations of the read and write cache are also wrapped in the file lock.

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [x] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: kclvm/runner/src/tests.rs [hunk 1]
```diff
@@ -212,6 +233,8 @@ fn test_kclvm_runner_execute_timeout() {
     match result_time_out {
         Err(panic_err) => {
             if let Some(s) = panic_err.downcast_ref::<String>() {
+                println!("{:?}", s);
```
Comment (Peefy) [inline kclvm/runner/src/tests.rs]: remove un-used code.

File: kclvm/runner/src/assembler.rs [hunk 2]
```diff
@@ -447,14 +408,14 @@ impl KclvmAssembler {
...
         lib_paths
     }
 }
 
-impl Default for KclvmAssembler {
-    #[inline]
-    fn default() -> Self {
-        Self::new()
+#[inline]
+pub fn clean_path(path: &str) {
```
Comment (Peefy) [inline kclvm/runner/src/assembler.rs:417]: pub -> pub(crate)

Comment (zong-zhe) [reply kclvm/runner/src/assembler.rs]: fixed

File: kclvm/runner/src/tests.rs (hunk 1 above)
Comment (zong-zhe) [reply kclvm/runner/src/tests.rs]: fixed

**Review by Peefy — APPROVED** [review]
LGTM

**Review by He1pa — APPROVED** [review]
lgtm

**Pull request merged**